code/modules/projectiles/projectile/bullets/junk.dm

```diff
@@ -0,0 +1,70 @@
+// Junk (Pipe Pistols and Pipeguns)
+
+/obj/projectile/bullet/junk
+	name = "junk bullet"
+	icon_state = "trashball"
+	damage = 30
+	embedding = list(embed_chance=15, fall_chance=3, jostle_chance=4, ignore_throwspeed_threshold=TRUE, pain_stam_pct=0.4, pain_mult=5, jostle_pain_mult=6, rip_time=10)
+	var/bane_multiplier = 1.5
+	var/bane_added_damage = 0
+
+/obj/projectile/bullet/junk/shock/Initialize(mapload)
```

Comment:
Can you move this next to its type declaration?

Reply:
I'm daft as fuck this is meant to be on all subtypes. I'll fix